feat: fix the README

Ayush7614 opened the pull request:
This PR Contain

* Fixed some typos and simplified some of the complex phrases.

Ayush7614 commented:
cc : @quozl @Raeskaa

Ayush7614 commented:
In the repo the in Aims in README all t are small it looks not good so i will fix to capital and added some information

Raeskaa reviewed (approved):
Looks good!

quozl commented:
Thanks, but I disagree with each of the changes.

The main problem with the README.md is as yet unaddressed; it contains far too much of one individual's contribution to rewriting the site, and not enough of the minimal information needed for ongoing development.  This in turn is because we have no maintainer.

Raeskaa commented:
> Thanks, but I disagree with each of the changes.
> 
> The main problem with the README.md is as yet unaddressed; it contains far too much of one individual's contribution to rewriting the site, and not enough of the minimal information needed for ongoing development. This in turn is because we have no maintainer.

I agree
The whole Readme.md must be updated and shortened, focusing on ongoing developments and updates 
excess information in the readme just makes it difficult for anyone who visits the repo.

Ayush7614 commented:
@quozl @Raeskaa You Guys Don't Worry I am planning for update the Sugarlabs website whole Readme yes I see the whole Readme and thinks are not in Proper Way I have full idea of development and Doing Changes properly in my NEXT few PRs

Ayush7614 commented:
> Thanks, but I disagree with each of the changes.
> 
> The main problem with the README.md is as yet unaddressed; it contains far too much of one individual's contribution to rewriting the site, and not enough of the minimal information needed for ongoing development. This in turn is because we have no maintainer.

If you don't mind I am maintaining this thing If you have no maintainer and I am address all the things in the README i am also setup local environment in My desktop for better understanding of Development how code works

Ayush7614 commented:
@quozl @Raeskaa You can merge this And in next PR I am doing some more Good changes and Contribution and Makes README verry helpful

Ayush7614 commented:
This is just the starting of the PR

quozl commented:
Thanks for your additional comments.

However, looking specifically at your pull request changes and comments relating to it;

> * Fixed some typos and simplified some of the complex phrases.

Your change from "web site" to "website" is not necessary and I don't think it simplifies or helps.  You can see in [Wikipedia - Website](https://en.wikipedia.org/wiki/Website) how "web site" is an alternate form.

You did not simplify any complex phrases.

> * Aims in README all t are small it looks not good so i will fix to capital and added some information

I think they look fine.  The added sentence is not needed, as the concept is implied in the existing heading and repository name.

Please do proceed to make further suggestions.  Please also make sure you use or develop one of our software products; Sugar, Sugarizer, or Music Blocks.  They are much more important than our web site, and are a pre-requisite for the necessary business of maintaining the content of the web site.

Ayush7614 commented:
> Thanks for your additional comments.
> 
> However, looking specifically at your pull request changes and comments relating to it;
> 
> > * Fixed some typos and simplified some of the complex phrases.
> 
> Your change from "web site" to "website" is not necessary and I don't think it simplifies or helps. You can see in [Wikipedia - Website](https://en.wikipedia.org/wiki/Website) how "web site" is an alternate form.
> 
> You did not simplify any complex phrases.
> 
> > * Aims in README all t are small it looks not good so i will fix to capital and added some information
> 
> I think they look fine. The added sentence is not needed, as the concept is implied in the existing heading and repository name.
> 
> Please do proceed to make further suggestions. Please also make sure you use or develop one of our software products; Sugar, Sugarizer, or Music Blocks. They are much more important than our web site, and are a pre-requisite for the necessary business of maintaining the content of the web site.

Okk Thanks i make some more changes

Raeskaa commented:
> Thanks for your additional comments.
> 
> However, looking specifically at your pull request changes and comments relating to it;
> 
> > * Fixed some typos and simplified some of the complex phrases.
> 
> Your change from "web site" to "website" is not necessary and I don't think it simplifies or helps. You can see in [Wikipedia - Website](https://en.wikipedia.org/wiki/Website) how "web site" is an alternate form.
> 
> You did not simplify any complex phrases.
> 
> > * Aims in README all t are small it looks not good so I will fix to capital and added some information
> 
> I think they look fine. The added sentence is not needed, as the concept is implied in the existing heading and repository name.
> 
> Please do proceed to make further suggestions. Please also make sure you use or develop one of our software products; Sugar, Sugarizer, or Music Blocks. They are much more important than our website and are a prerequisite for the necessary business of maintaining the content of the website.

@quozl 
we can make another fille which will contain detailed information about the website ( which is currently on the Readme.md ) and try to make the updated readme file more concise and to the point

Ayush7614 commented:
> > Thanks for your additional comments.
> > However, looking specifically at your pull request changes and comments relating to it;
> > > * Fixed some typos and simplified some of the complex phrases.
> > 
> > 
> > Your change from "web site" to "website" is not necessary and I don't think it simplifies or helps. You can see in [Wikipedia - Website](https://en.wikipedia.org/wiki/Website) how "web site" is an alternate form.
> > You did not simplify any complex phrases.
> > > * Aims in README all t are small it looks not good so I will fix to capital and added some information
> > 
> > 
> > I think they look fine. The added sentence is not needed, as the concept is implied in the existing heading and repository name.
> > Please do proceed to make further suggestions. Please also make sure you use or develop one of our software products; Sugar, Sugarizer, or Music Blocks. They are much more important than our website and are a prerequisite for the necessary business of maintaining the content of the website.
> 
> @quozl we can make another fille which will contain detailed information about the website ( which is currently on the Readme.md ) and try to make the updated readme file more concise and to the point

@Raeskaa I am update full Readme dont worry

Ayush7614 commented:
cc @quozl

Ayush7614 commented:
@quozl can you tell me i am removing the history part and all the gsoc 2017 history and all
![image](https://user-images.githubusercontent.com/67006255/152732134-a476b1d7-3533-483c-923f-b92e88c66b9e.png)

Ayush7614 commented:
![image](https://user-images.githubusercontent.com/67006255/152732253-ddf4a260-1b8d-4781-a192-242796387bdd.png)

Ayush7614 commented:
Like this all parts we can remove this one and shifted in one seperate file what was the idea? @quozl @Raeskaa

Ayush7614 commented:
Because from all these readme looks not good

Ayush7614 commented:
cc @quozl now you can see

quozl commented on README.md:
```diff
@@ -1,28 +1,88 @@
 # [Sugar Labs](https://www.sugarlabs.org/)
-# Sugar Labs web site
-    
+# Sugar Labs Website
```
Unnecessary change.

quozl commented on README.md:
```diff
@@ -1,28 +1,88 @@
-    
+# Sugar Labs Website
+
+This repository provides information about the Sugar Labs organization official website. 
```
Unnecessary change.

quozl commented on README.md:
```diff
@@ -1,28 +1,88 @@
+* To invite potential users of Sugar Labs software,
+* To advertise our free software and licensing approach,
+* To provide links to software downloads, and;
+* To invite potential developers to the source code we hold in trust.
```
Unnecessary change.

quozl commented on README.md:
```diff
@@ -1,28 +1,88 @@
+## Introduction
+
+[Sugar Labs](https://www.sugarlabs.org/) is a community-run software project whose mission is to produce, distribute, and support the use of Sugar, an open source desktop environment and learning platform.. The Main software is the [Sugar](https://github.com/sugarlabs/sugar), Sugar is the desktop environment component of a worldwide effort to provide every child with an equal opportunity for a qu ...
+
```
While true, it doesn't need to be said in our web site repository.

quozl commented on README.md:
````diff
@@ -1,28 +1,88 @@
+  git checkout main
+  git fetch upstream
+  git merge upstream/main
+  ```
````
Not necessary, this is all normal GitHub workflow, better to point to GitHub documentation.

quozl commented on README.md:
```diff
@@ -1,28 +1,88 @@
-## Contributing
+## How to Get Involved and Contribute?
 
-Content on the site; that is information about Sugar, Sugarizer, or Music Blocks, is _way more important_ than the style or appearance.
```
Unwelcome change, please keep this line.

quozl commented on README.md:
```diff
@@ -1,28 +1,88 @@
-Content on the site; that is information about Sugar, Sugarizer, or Music Blocks, is _way more important_ than the style or appearance.
+Sugar Labs welcomes contributions to the Sugar Labs official website and all other Projects. If you have an idea for a new feature or a bug fix, please submit an issue or pull request.
 
-Please concentrate on updating the content of the site, and not the style or appearance.
```
Unwelcome change, please keep this line.

quozl commented on README.md:
```diff
@@ -1,28 +1,88 @@
-Please concentrate on updating the content of the site, and not the style or appearance.
+The open source community is the heart of Sugar Labs and helps us build tools of the Sugar Labs organization. Some of the our important projects in which you can start contributing are :
 
-For your pull requests or issues to be taken seriously, you must be a user or developer of one of our software products; Sugar, Sugarizer, or Music Blocks.
```
Unwelcome change, please keep this line.

quozl commented on README.md:
```diff
@@ -67,33 +127,6 @@ Base template has been designed and developed by Themefisher. For a preview, che
-Then open [http://localhost:4000](http://localhost:4000)
-### Note
-If some changes are not visible (after saving a page while editing), please restart the Jekyll local web server and reload [http://localhost:4000](http://localhost:4000).
-
```
In the end, all you have done is removed the policies you disagree with, and move things around in the file.

quozl reviewed (changes requested):
@Ayush7614, before I read anything of what you said, please be considerate of the 9 notifications you have directed at our 24 repository followers.  That's 216 notifications.  Please compose your response in full before pressing submit.  Expect a response from other members within a couple of days.

Your commit messages are practically empty.  Please rewrite your commit messages.  See the [Making Commits](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits) section of our guide for contributors.  Commit messages are where description and justification of your changes should be written.

You should also read our contributing guide in full.  It may help with my next comments that you read the [Guide for Reviewers](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#guide-for-reviewers)

Looking at the [Files changed](https://github.com/sugarlabs/www-sugarlabs/pull/372/files) tab for the pull request, my review comments are;

* [ ] again, there's no need to change "web site" to "website",
* [ ] again, there's no need to change the initial letter to uppercase in the Aims section,
* [ ] again, line 10 says something with is so obvious it doesn't need to be said,
* [ ] do not remove the policy sections we have set for the repository; these have evolved over several years as people try to repaint the bike shed instead of fix the bikes,

Ayush7614 commented:
Sorry @quozl I am never doing this again now i make changes please review it. 
This PR Contains
Update in the installation guide, Add all the rules of installation
Technology used in the website and organization
Adding poster of the Sugar labs in the Readme
Adding how to get involved and contributions in main repos and projects of sugar labs
Remove unnecessary things

quozl commented:
I shall defer my review until I've heard from other Sugar Labs members.  While you wait, you're welcome to go through the _Files changed_ tab to make sure you're proposing every change that GitHub has recorded, and rewrite your git commit messages (not pull request messages).

chimosky commented:
Hey @Ayush7614, I've reviewed your changes and I agree with quozl as you've made some changes based on your preferences while the things you're changing were discussed and agreed on by various members of the community but none of your changes have been discussed and I don't agree with your changes.

quozl commented:
Thanks @chimosky.  @Ayush7614, I shall await your edits to satisfy the review comments.

Ayush7614 commented:
> Thanks @chimosky. @Ayush7614, I shall await your edits to satisfy the review comments.

@quozl @chimosky Firstly Thanks for Reviewing my PR Can you suggest me what changes i do in this PR

quozl commented:
Checked discussion; no response to feedback, closing.
